pkgs/shells/bash/5.nix

```diff
@@ -91,7 +91,7 @@ stdenv.mkDerivation rec {
 
   strictDeps = true;
   # Note: Bison is needed because the patches above modify parse.y.
-  depsBuildBuild = [ buildPackages.stdenv.cc ];
+  depsBuildBuild = [ stdenv ];
```

Comment:
I don't think this will work? how would `stdenv` become `stdenv.cc`?

Reply:
You're right; this works "by accident".